lib/__tests__/cookieToAndFromJson.spec.ts

```diff
@@ -1,15 +1,20 @@
+import { describe, expect, it } from 'vitest'
 import { Cookie } from '../cookie/cookie.js'
 
```

Comment:
The Jest fake timers were [enabled globally](https://github.com/salesforce/tough-cookie/blob/master/jest.config.ts#L9) since we often need finer control over date-based testing. It might be worthwhile to do something similar for `vitest` but, barring that, adding the following will help to resolve the need for capturing and comparing a `[start, end]` date for these tests which did fail several times for me locally.

```suggestion
import { beforeEach, describe, expect, it, vi } from "vitest";
import { Cookie } from "../cookie/cookie.js";

beforeEach(() => {
  vi.useFakeTimers()
})
```